atomkraft/reactor/reactor.py

```diff
@@ -73,8 +73,8 @@ def get_reactor() -> PathLike:
 def generate_reactor(
     actions_list: List[str],
     variables_list: List[str],
+    stub_file_path: PathLike,
     keypath: str = "action",
-    stub_file_path: Optional[PathLike] = None,
```

Comment:
In Python, you should be able to do `PathLike | None`.

Doesn't it work here?

Reply:
You are commenting on the old code. I actually removed `Optional` because I want this parameter to be mandatory.